examples/xonotic/Makefile

```
@@ -23,12 +23,17 @@
 #     \_/ \__,_|_|  |_|\__,_|_.__/|_|\___|___/
 #
 
-REPOSITORY = us-docker.pkg.dev/agones-images/examples
+REPOSITORY ?=
+PROD_REPO ?= us-docker.pkg.dev/agones-images/examples
 
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 project_path := $(dir $(mkfile_path))
 root_path := $(realpath $(project_path)/../..)
-image_tag = $(REPOSITORY)/xonotic-example:1.1
+ifeq ($(REPOSITORY),)
+	image_tag := xonotic-example:1.1
+else
+	image_tag := $(PROD_REPO)/xonotic-example:1.1
```

So my original thought was to do the following:

```suggestion
	image_tag := $(REPOSITORY)/xonotic-example:1.1
```

That would mean that `echo-image-tag` would need then to be:

```Makefile
echo-image-tag:
	@echo $(PROD_REPO)/$(image_tag)
```

Which I think is fine? WDYT?

---

updated it as per your suggestion